misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/TesterinaConstants.java

```diff
@@ -101,4 +101,18 @@ public class TesterinaConstants {
     public static final int IDENTIFIER_START_INDEX = 1;
     public static final int IDENTIFIER_END_INDEX = 5;
 
+    //class to identify the indices of the BTestMain args
```

Comment:
Since this is a public class, let's have a documentation comment similar to TesterinaConstants class.

```
/**
 * RuntimeArgs identifies the indices of the BTestMain arg
 *
 * @since 2201.9.0
 */
```

Reply:
Added in 9fe35ce commit